slack_sdk/models/blocks/block_elements.py

```diff
@@ -514,6 +481,63 @@ def _validate_initial_time_valid(self) -> bool:
         return self.initial_time is None or re.match(r"([0-1][0-9]|2[0-3]):([0-5][0-9])", self.initial_time) is not None
 
 
+# -------------------------------------------------
+# DateTimePicker
+# -------------------------------------------------
+
+
+class DateTimePickerElement(InputInteractiveElement):
+    type = "datetimepicker"
+
+    @property
+    def attributes(self) -> Set[str]:
+        return super().attributes.union({"initial_date_time"})
+
+    def __init__(
+        self,
+        *,
+        action_id: Optional[str] = None,
+        initial_date_time: Optional[int] = None,
+        confirm: Optional[Union[dict, ConfirmObject]] = None,
+        focus_on_load: Optional[bool] = None,
+        **others: dict,
+    ):
+        """
+        An element that allows the selection of a time of day formatted as a UNIX timestamp.
+        On desktop clients, this time picker will take the form of a dropdown list and the
+        date picker will take the form of a dropdown calendar. Both options will have free-text
+        entry for precise choices. On mobile clients, the time picker and date
+        picker will use native UIs.
+        https://api.slack.com/reference/block-kit/block-elements#datetimepicker
+
+        Args:
+            action_id (required): An identifier for the action triggered when a time is selected. You can use this
+                when you receive an interaction payload to identify the source of the action. Should be unique among
+                all other action_ids in the containing block. Maximum length for this field is 255 characters.
+            initial_date_time: The initial date and time that is selected when the element is loaded, represented as
+                a UNUIX timestamp in seconds. This should be in the format of 10 digits, for example 1628633820
```

Comment:
I've reported this error and it's already fixed on the original document side
```suggestion
                a UNIX timestamp in seconds. This should be in the format of 10 digits, for example 1628633820
```